osh/hub/osh_xmlrpc/worker.py

```diff
@@ -27,6 +27,28 @@
 
 logger = logging.getLogger(__name__)
 
+# DO NOT REMOVE!  The __all__ list contains all publicly exported XML-RPC
+# methods from this module.
+__all__ = [
+    'cancel_task',
+    'create_sb',
+    'email_scan_notification',
+    'email_task_notification',
+    'ensure_base_is_scanned_properly',
+    'ensure_cache',
+    'fail_scan',
+    'fail_task',
+    'finish_analyzers_version_retrieval',
+    'finish_scan',
+    'finish_task',
+    'get_scanning_args',
+    'get_su_user',
+    'interrupt_tasks',
+    'move_upload',
+    'set_scan_to_basescanning',
+    'set_scan_to_scanning',
```

Comment:
I confirmed that some of the methods are indeed gone with this patch, e.g., the `worker.AnalyzerVersion`. However I still see some of inconsistencies, i.e., I see around 34 "worker.xxx" methods under `/xmlrpc/worker/` while here I see only 17.

For example I see `worker.create_subtask`/`worker.assign_task` etc.

Reply:
The default set of XML-RPC methods is exported by `kobo`: https://github.com/release-engineering/kobo/blob/master/kobo/hub/xmlrpc

OSH either adds new methods or overrides the ones from `kobo` to add some extra functionality.  The set of all exported Python modules is defined here: https://github.com/openscanhub/openscanhub/blob/ac9459c863ecc62bcd135f7db18b56a311c0c020/osh/hub/settings.py#L152-L178

Reply:
aha I see, thanks